gz_ros2_control_demos/config/diff_drive_controller.yaml

```diff
@@ -36,19 +36,17 @@
     # Whenever a min_* is unspecified, default to -max_*
     linear.x.has_velocity_limits: true
```

Comment:
velocity and acceleration are deprecated similarly (just no output if the value is true, i.e., the default)